.github/workflows/unit-tests.yaml

```diff
@@ -15,12 +15,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [ '3.0', '3.0.4', '3.1.0', '3.2.0' ]
+        ruby-version: [ '3.1.0', '3.2.0', '3.3.0' ]
```

Comment:
do we need to remove `3.0` support?

Reply:
3.0 is EOL https://endoflife.date/ruby

https://www.ruby-lang.org/en/downloads/

Reply:
only released 3.5 years ago... I assume a lot of folks are still using it.

Reply:
We don't support EOL versions in any other SDK.